source/db/en-projects.json

```diff
@@ -714,7 +714,7 @@
     "description": "Conversations is an open source XMPP/Jabber client for Android 4.0+ smart phones.",
     "license_url": "https://github.com/siacs/Conversations/blob/development/LICENSE",
     "logo": "conversations.png",
-    "notes": "",
+    "notes": "Conversations' one native code dependency on iNPUTmice is [discussed here](https://github.com/nylira/prism-break/issues/1659#issuecomment-269921269).",
```

Comment:
Do other localizations have a `notes` key for Conversations? If so they need to be removed.

Reply:
@strugee If I understand this the commit 226d8a4  on my #1854 PR will take care of removing all blank note fields across all listings. If that's not correct then I'm not quite sure I understand what's going on.

Reply:
@alerque I was wondering if there's a `notes` field for other Conversations listing that somehow got out of sync and had text in it (and therefore wasn't picked up by your commit). I would assume not but it'd be nice to check. Seems pretty unlikely though so if you don't have time to check I'd be fine merging this as-is :thumbsup:

Reply:
@strugee I just ran a query on that field and there are no other `.notes` keys with obsolete content on Conversations listings in any language, so no ripping out just the blanks didn't miss any in this case. Good thought though.

Reply:
Great, thanks for checking! LGTM!